Documentation/gettingstarted/minikube-install.rst

```diff
@@ -22,11 +22,18 @@
 
      minikube start --network-plugin=cni --memory=4096
 
+.. note::
+
+   If minikube is deployed as a container (that is if ``docker`` is the configured
+   driver), then some of the kube-proxy replacement features may not work (`GitHub
```

Comment:
Let's maybe be precise about which feature is not going to work (i.e. host-reachable svc).